tests.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+require './decorator'
```

Comment:
- [ ] [OPTIONAL] Please consider using `require_relative` instead of `require` whenever you are importing a part of your code which resides in another file in your project. `require` should only be used when using an external dependency 👍 
- [ ] [OPTIONAL] I understand that the `tests.rb` file is not a requirement; you tried to make life easier to test the code by that file so this comment will be optional. Don't forget to require the below files so that your `tests.rb` can run without a hitch 👇 
```ruby
require './trimmer_decorator'
require './caapitalize_decorator'
```